README.rst

```diff
@@ -0,0 +1,153 @@
+=======================
+EOX core |build-status|
+=======================
+
+.. |build-status| image:: https://circleci.com/gh/eduNEXT/eox-core.svg?style=svg
+
+Eox-core (A.K.A. Edunext Open eXtensions) is a `django app plugin](https://github.com/edx/edx-platform/tree/master/openedx/core/djangoapps/plugins) for adding multiple APIS for the [edx-platform <https://github.com/edx/edx-platform>`_ for adding multiple APIS for the `edx-platform <https://github.com/edx/edx-platform>`_ to make changing the platform easier on commercial environments, including bulk creation of pre-activated users (e.g. skip sending an activation email) and enrollments.
+
+Usage
+=====
+
+1) Create the oauth client at http://localhost:18000/admin/oauth2/client/add/ , copy the client-id and client-secret.
+
+2) Generate an auth-token using that client-id and client-secret:
+
+.. code-block:: bash
+
+	$ curl -X POST -d "client_id=<YOUR_CLIENT_ID>&client_secret=<YOUR_CLIENT_SECRET>
+		&grant_type=client_credentials" http://localhost:18000/oauth2/access_token/
+
+3) Use the token to call the API as you need:
+
+.. code-block:: bash
+
+	# User creation API example
+	curl -X POST --header "Authorization: Bearer <YOUR_AUTH_TOKEN>" -H "Accept: application/json" \
+		http://localhost:18000/eox-core/api/v1/user/ --header "Content\-Type: application/json" \
+		--data  '{"username": "jsmith", "email": "jhon@example.com", "password": "qwerty123",
+		"fullname": "Jhon Smith"}'
+
+	# Enroll api example
+	curl -X POST --header "Authorization: Bearer <YOUR_AUTH_TOKEN>" -H "Accept: application/json" \
+		http://localhost:18000/eox-core/api/v1/enrollment/ --header "Content-Type: application/json" \
+		--data '{"course_id": "course-v1:edX+DemoX+Demo_Course", "email": "edx@example.com",
+		"mode": "audit", "force": 1}'
+
+Installation on Devstack
+========================
+- Create an openedx Ironwood instace if it doesn't exist already:
```

Comment:
Is this necessary? Why not just add the link to the devstack instructions? @morenol